### PR TITLE
Add Structured data of organisation

### PR DESCRIPTION
**Pull request by nstapelbroek**
Hey Rick,

Awesome site :beers:  Would you be okay if we tried to display the content (social links) as structured data? I think this will result in a custom block[ when Googling your company name](https://developers.google.com/search/docs/data-types/social-profile). 

If you have other plans for this kind of stuff feel free to close the PR. no worries.

Have a good one :tada:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,10 +37,22 @@
             background-image: url(images/email.svg);
         }
     </style>
+    <script type="application/ld+json">
+    {  
+       "@context":"http://schema.org",
+       "@type":"Organization",
+       "url":"https://ngplusdevelopment.com",
+       "logo":"https://ngplusdevelopment.com/images/logo.svg",
```
Comment (nstapelbroek) [inline index.html:45]: :thinking: I don't think they are going to add a background to the logo. Do you have something that works on a white background?

**Comment by rskuipers** [comment]
You're fucking awesome, Nico!

File: index.html (hunk 1 above)
Comment (rskuipers) [reply index.html:45]: I don't yet but I will make a square one with a proper background, good tip!

**Comment by rskuipers** [comment]
Thanks man!

**Comment by rskuipers** [comment]
Seems to validate properly: https://search.google.com/structured-data/testing-tool/u/0/#url=https%3A%2F%2Fngplusdevelopment.com%2F

**Comment by nstapelbroek** [comment]
Yea, I was also peeking :see_no_evil: Submitted a small fix in #2

**Pull request merged**